src/ardrive.ts

```diff
@@ -528,10 +528,22 @@ export class ArDrive extends ArDriveAnonymous {
 
 		const conflictingFileName = filesAndFolderNames.files.find((f) => f.fileName === destFileName);
 
-		if (conflictingFileName && conflictResolution === 'skip') {
-			return emptyArFSResult;
+		if (conflictingFileName) {
+			if (conflictResolution === 'skip') {
```

Comment:
Can we get some constants for these flag values?

Reply:
Added: c1f8da0